configure.ac

```diff
@@ -1898,6 +1898,9 @@
         fi
     fi
 
+# Check for lz4
+AC_CHECK_LIB(lz4, LZ4F_createCompressionContext, , )
```

Comment:
it would be nice to give user a hint about what .deb or .rpm to install, similar to most other dependencies